### PR TITLE
Release Google.Cloud.Profiler.V2 version 1.0.0

### PR DESCRIPTION
**Pull request by jskeet**
Changes in this release:

No API surface change since 1.0.0-beta01

### CODE REVIEW

**Review by amanda-tarafa — APPROVED** [review]
LGTM

**Comment by jskeet** [comment]
(Will wait until the Memcache release has finished before merging this...)

**Pull request merged**
